data/map systems.txt

```diff
@@ -31145,6 +32988,50 @@ system Sumprast
 			distance 464
 			period 29.291
 
+system Tabit
+	pos -265.33333 126.33334
+	government Syndicate
+	link Acrab
+	link Iklil
+	asteroids "small rock" 21 0.53
+	asteroids "medium rock" 254 0.61
+	asteroids "large rock" 5 0.58
+	asteroids "large metal" 1 0.65
+	minables neodymium 9 0.76902448
+	minables lead 35 1.0955883
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Pirates" 1000
	fleet "Large Core Pirates" 2500
	fleet "Small Core Merchants" 2000
```
Exactly the same fleets as Iklil, since any traffic through one must also be going through the other.